prov/rxd/src/rxd_ep.c

```diff
@@ -1042,14 +1042,29 @@ static void rxd_buf_region_free_fn(struct ofi_bufpool_region *region)
 		fi_close(region->context);
 }
 
+static int rxd_pool_create_attrs(struct rxd_ep *ep, struct rxd_buf_pool *pool,
+					struct ofi_bufpool_attr attr,
+					enum rxd_pool_type type)
+{
+	int ret;
+	pool->rxd_ep = ep;
+	pool->type = type;
+
+	ret = ofi_bufpool_create_attr(&attr, &pool->pool);
+	if (ret)
+		FI_WARN(&rxd_prov, FI_LOG_EP_CTRL,
+			"Unable to create buf pool\n");
+	return ret;
+
```

Comment:
nit: extra blank line.  I'm okay with this patch.  But please drop the other patch and keep the container_of() call directly in the code.

Reply:
@shefty Thanks for nit and review! I have dropped unnecessary changes. Now the patch consists of providing `rxd_pool_create_attrs()` function in order to summarize `ofi_bufpull_create_attr(`) for `*pool_create` ones.